crest/Assets/Crest/Crest/Scripts/LodData/OceanDepthCache.cs

```diff
@@ -39,6 +40,58 @@ public class OceanDepthCache : MonoBehaviour
         GameObject _drawCacheQuad;
         Camera _camDepthCache;
 
+        public static bool UseGeometryShader
```

Comment:
is UseGeometryShader specific to OceanDepthCache or is it now used by multiple lod types? If the latter we could put it perhaps in LodDataMgr..